src/pages/index.md

```diff
@@ -15,6 +15,29 @@ Here's a sample:
 )
 ```
 
+## Why Lisp?
+
+Many people come into our keybase channel and ask us why we chose a 60 year old language as our on chain programming language.
+We chose it due to a few unique features that make it remarkably well suited to the Chia blockchain:
+
+* **Completely sandboxed.** Chialisp has *absolutely no side effects*.  The language needs to be run on half a million computers, so it is important that the program cannot reach out and affect everyone's machines in an unintended way.  A lisp program is *evaluated* and therefore cannot spawn any new processes or interact with the system it is running on.
+
+* **Programs are data structures.** A lisp program is itself just a list.  This allows for powerful techniques that allow you to modify source code during program evaluation.  Doing so can allow a "smart contract" to enforce rules on a participating coin while still allowing it to utilize the full programmability that Chialisp has to offer.  Using lisp programs like this allows you to have *layers of smart contracts*.
+
+* **Interoperability.** There is no typing in lisp, everything becomes a sequence of bytes that is only interpreted differently by choice of the operation that is reading/manipulating it.  The means that everything in the Chia ecosystem interoperates with everything else.  Any smart contract should be able to interact or communicate with any other coin or smart contract, regardless of whether either smart contract was designed to do so.
```

Comment:
I'm not sure the points in this paragraph all refer to what's generally considered "interoperability". Also, I prefer the word "composability" to emphasize that a CL contract takes a "solution" of any kind as input and generally produces "conditions" as output... which means that you can create puzzles that expect their "solution" to also include conditions produced by an inner puzzle. "Composability" means nesting inner puzzles, to an arbitrary depth (or at least, a depth not directly limited by CL).